ChangeLog

```diff
@@ -2876,6 +2876,10 @@ Release date: 2003-09-12
 
     * easy functional test infrastructure
 
+	* Raise bad-whitespace error on arithmetic operators that miss spaces
```

Comment:
Can you check the indentation here? Seems to use tabs or it has too many spaces. Also, this uses the wrong section for the release, it should be the latest.